corehq/apps/ota/views.py

```diff
@@ -63,8 +63,8 @@ def search(request, domain):
                  .case_type(case_type)
                  .is_closed(False)
                  .size(CASE_SEARCH_MAX_RESULTS))
-    config = CaseSearchConfig(domain=domain).config
-    fuzzies = config.get_fuzzy_properties_for_case_type(case_type)
+    config, _ = CaseSearchConfig.objects.get_or_create(domain=domain)
```

Comment:
In theory they should already have a `CaseSearchConfig` object, otherwise there would be no way of getting the case search option on the phone. I suppose I could have done `get` here and failed hard if this wasn't the case, but that seemed like bad UX. Do you have other suggestions?